Tools/cases_generator/stack.py

```diff
@@ -2,7 +2,7 @@
 from analyzer import StackItem, StackEffect, Instruction, Uop, PseudoInstruction
 from dataclasses import dataclass
 from cwriter import CWriter
-from typing import Iterator
+from typing import Dict, Iterator, Tuple
```

Comment:
Do you need Python 3.6+ compatibility or can you live with `dict` and `tuple`? (I did not add suggestions in case you don't want to but the tier generator files should be changed in case you want to use `dict[K, V]` and `tuple[V, ...]` syntax).